cypress/fixtures/successful_signup.json

```diff
@@ -0,0 +1,10 @@
+{
+  "data": {
+    "id": 1,
+    "role": "user",
+    "email": "user@mail.com",
+    "provider": "email",
+    "uid": "user@mail.com",
+    "allow_password_change": false
+  }
+}
```

Comment:
Maybe uneccasary to add this fixture, but was unsure if it was the same as login so that is why....